res/css/views/dialogs/_LocationViewDialog.scss

```diff
@@ -49,8 +49,48 @@ limitations under the License.
         }
     }
 
-    .mx_MLocationBody .mx_MLocationBody_map {
-        width: 80vw;
-        height: 80vh;
+    .mx_MLocationBody {
+        position: absolute;
+
+        .mx_MLocationBody_map {
+            width: 80vw;
+            height: 80vh;
```

Comment:
viewport sizing is something we don't typically use - if it works, it works, but just a heads up.

Reply:
Thank yeah, I noticed that, but it was the only way I could make it work in this instance. It took quite a lot of fiddling to get this to look right.